members/_posts/2017-08-21-faps-ehome.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: memberpage
+category: cmember
+logo : faps-ehome.png
+
+name: 'E|Home-Center - Bayerisches Technologiezentrum für privates Wohnen'
+address1: 'Fürther Straße 246b, 90429 Nürnberg'
+address2: 
+address3: 
+address4: 
+city: 'Nürnberg'
+country: 'Germany'
+phone1: '+49 911 5302 9083'
+email1: anja.stoeckigt@faps.fau.de
+phone2: '+49 911 5302 9112'
+email2: jochen.bauer@faps.fau.de
+web: http://www.ehome-center.de
+twitter: 
+github:
+
+---
+The E|Home-Center develops and researches technologies for the intelligent, energy-efficient and resource-efficient housing in the private sector.
+
+The interdisciplinary research is carried out in cooperation with other research institutions, private companies and other relevant organizations. At the E|Home-Center, scientists from different disciplines work closely together. Engineers, computer scientists and social scientists develop concepts for securing an efficient energy supply and use as well as for creating a self-determined, age-appropriate and individual living space in the private residential environment. Research is focused on intelligent systems and devices that work autonomously and interoperably, and enable people to use intuitive operation, including using speech or gestures.
```

Comment:
Could you use only one sentence per line please? Even if that file should not change that much, it keeps things more readble on the source level.